lumigator/python/mzai/sdk/tests/data/job-all.json

```diff
@@ -4,8 +4,8 @@
     "model": "some_model",
     "dataset": "6f6487ac-7170-4a11-af7a-0f6db1ec9a74",
     "max_samples": 5,
-    "model_url": "s3://some_url",
+    "model_url": "hf://facebook/bart-large-cnn",
```

Comment:
In theory `model` should contain the model identifier, `model_url` the actual URL the model is served from (just in the case of inference APIs), and the two can coexist but `model_url` can also be null.

You can change this with an actual URL if you want, but I think this data file (or its corresponding SDK test) is not correctly capturing our format. I also see there are still `config_*_template` instead of the single `config_template` parameter but no error was raised during SDK tests so I [opened an issue ](https://github.com/mozilla-ai/lumigator/issues/393) for this. Feel free to ignore the issue for the sake of this PR!

Reply:
I'll change this to a URL for now and we can address the issue you opened-thanks fro that!